src/pages/_error.tsx

```diff
@@ -1,23 +1,98 @@
-import React from "react";
+import React, { useContext, useEffect } from "react";
 import { NextPageContext } from "next";
 import Layout from "../components/Layout/Layout";
 import Link from "../components/Link/Link";
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Image,
+  Text,
+} from "@nypl/design-system-react-components";
+import { FeedbackContext } from "../context/FeedbackContext";
+
+const ERROR_PERSISTS = " if the error persists.";
+
+const getNotificationText = (statusCode) => {
+  return `You are asking for help or information about a page error (error code ${statusCode})`;
+};
+
+const errorMap = {
```

Comment:
Should we also add a 403 mapping for limited access?

Reply:
403 is not returned for the limited access routes. For invalid auth, the user is redirected to the login page.